src/client/sse-manager.ts

```
@@ -29,24 +34,28 @@ class SSEManager {
     if (connection) {
-      if (!connection.listeners.has(eventName)) {
-        connection.listeners.set(eventName, new Set());
-        connection.source.addEventListener(eventName, (event) => {
-          const listeners = connection.listeners.get(eventName);
-          listeners?.forEach((listener) => listener(event));
-        });
+      let entry = connection.listeners.get(eventName);
+      if (!entry) {
+        const forwarder: EventListener = (event) => {
+          const entry = connection.listeners.get(eventName);
+          entry?.userListeners.forEach((listener) => listener(event as MessageEvent));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)